script/main.js

```diff
@@ -203,4 +203,29 @@ const projectDesktopButton5 = document.querySelector('#seeproject5');
 projectDesktopButton5.addEventListener('click', dynamicModalDesktop2);
 
 const projectDesktopButton6 = document.querySelector('#seeproject6');
-projectDesktopButton6.addEventListener('click', dynamicModalDesktop3);
\ No newline at end of file
+projectDesktopButton6.addEventListener('click', dynamicModalDesktop3);
+
+const emailElement = document.querySelector('#email');
+const emailValid = function checkEmail(email) {
+  const regexp = /^[a-z]+(([a-z]+\.?([0-9]+)?\.?))+([a-z0-9])@([a-z]+\.?)+[a-z]{2,}$/;
+  return regexp.test(email);
+};
+const isEmailValid = function validateEmail() {
+  let valid = false;
+  const email = emailElement.value.trim();
+  if (!emailValid(email)) {
+    document.querySelector('#error').innerHTML = '* Please enter a valid email.';
```

Comment:
- [ ] [optional] It would be nice to rewrite the error message to be more clear on the issue of why the email s not valid.